core/src/main/scala/yaml/ManifestV1Parser.scala

```diff
@@ -127,7 +127,7 @@ object ManifestV1Parser {
   }
 
   def validateInstances(i:Int): YamlValidation[Int] =
-    validatePlanConstraints(i, invalidInstances)(i => i > 0 && i < 500)
+    validatePlanConstraints(i, invalidInstances)(i => i > 0 && i < 5000)
```

Comment:
Is this just validating that the number of instances/replicas of a workload is 0 < i < 5000? Why have an upper bound at all?

Reply:
@adelbertc at the time we wanted a way to ensure that the number people specified was "sane", so that people didn't just ask for more and more replicas (like a million, for example). WDYT?